library/src/test/java/com/datatorrent/lib/appdata/datastructs/DimensionalTableTest.java

```diff
@@ -38,7 +38,7 @@ public void serializabilityTest() throws Exception
   {
     DimensionalTable<Integer> table = createTestTable();
 
-    TestUtils.clone(new Kryo(), table);
+    KryoCloneUtils.cloneByKryo(new Kryo(), table);
```

Comment:
IMO, should use `KryoCloneUtils.cloneByKryo(table);`and possible remove Kryo import.